src/templates/doe_workflow_attic_air_sealing.mdx

```diff
@@ -0,0 +1,291 @@
+<Tabs>
+<Tab eventKey="project" title="Project">
+    The data for this tab should be know prior to the initial site visit.
+
+    <ProjectInfoInputs {...props} />    
+  </Tab>
+  <Tab eventKey="assessment" title="Assessment">
+    The data for this tab should be collected during the initial site visit.
+     
+    ## Building 
+    <PhotoInput id="building_number_photo" label="Building Number – Photo">
+      Provide a photo of the building that shows the street number.
+    </PhotoInput>
+
+    ## Blower Door
+    <PhotoInput id="blower_door_setup" label="Blower Door Setup – Photo">
```

Comment:
All photo ids should end with "_photo" to match our earlier system.

Reply:
Update the id of all PhotoInput and Photo components. Tested the changes proper update in the database.